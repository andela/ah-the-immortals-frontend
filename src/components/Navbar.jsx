import React, { Component } from 'react';
import { Link } from 'react-router-dom';


export class Navigationbar extends Component {
  render() {
    return (
      <ul>
        <li>
          <Link to="/"> Home </Link>
        </li>
        <li>
          <Link to="/Login"> Dummy Login </Link>
        </li>
      </ul>
    );
  }
}

export default Navigationbar;

