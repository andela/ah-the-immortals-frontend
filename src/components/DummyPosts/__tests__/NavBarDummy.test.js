import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Navbar from '../NavBarDummy';

describe('Tests dummy navigation Bar', () => {
  it('Tests dummy navigation Bar component', () => {
    const wrapper = shallow(
      <Navbar />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
