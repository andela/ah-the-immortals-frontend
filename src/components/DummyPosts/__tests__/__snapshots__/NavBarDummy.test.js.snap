// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests dummy navigation Bar Tests dummy navigation Bar component 1`] = `
<div>
  <Bootstrap(undefined)
    className="homenav fixed-top"
  >
    <NavbarBrand>
      <b
        className="brand"
      >
        Authors Haven
      </b>
    </NavbarBrand>
    <NavbarCollapse>
      <Nav
        className="justify-content-end"
        fill={false}
        justify={false}
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <NavItem>
          <Link
            to="/profile"
          >
            <Button
              active={false}
              className="buttons prifile-button"
              disabled={false}
              type="button"
              variant="outline-light"
            >
              Profile
            </Button>
          </Link>
        </NavItem>
        <NavItem>
          <Button
            active={false}
            className="buttons"
            disabled={false}
            type="button"
            variant="outline-light"
          >
            Log out
          </Button>
        </NavItem>
      </Nav>
    </NavbarCollapse>
  </Bootstrap(undefined)>
</div>
`;
